ext/opentelemetry-ext-opentracing-shim/src/opentelemetry/ext/opentracing_shim/__init__.py

```diff
@@ -22,20 +88,56 @@
 
 
 def create_tracer(otel_tracer):
+    """Creates a :class:`TracerShim` object from the provided OpenTelemetry
+    :class:`opentelemetry.trace.Tracer`.
```

Comment:
FWIW the [`default_role = "any"` in sphinx config](https://github.com/kinvolk/opentelemetry-python/blob/bd020e9720f343a5a370cfb673df7beb4a98ac44/docs/conf.py#L32) means you can omit the role (`:class:` here) in most cases. This also goes for `:obj:` for built-in types.

I think the docstrings (in code, not rendered as html) are more readable without the roles, but up to you.

Reply:
Thanks, I've noticed this behavior and I agree regarding readability. I've added `:class:` etc. to be consistent with existing docs, for example https://github.com/open-telemetry/opentelemetry-python/blob/master/opentelemetry-api/src/opentelemetry/trace/__init__.py.

Reply:
The existing docs should probably change too. They were written before we changed the default role.